pennylane/tape/qscript.py

```diff
@@ -438,6 +449,25 @@ def _update_observables(self):
                     self._obs_sharing_wires.append(m.obs)
                     self._obs_sharing_wires_id.append(i)
 
+    def _update_measured_wires(self):
+        """Updates the measured wires straightforwardly.
+
+        Sets:
+            _measured_wires (int): The wires measured in this QuantumScript
+        """
+
+        def extract_wires(obs):
+            for wire in obs.wires:
+                if wire not in self._measured_wires:
+                    self._measured_wires.append(wire)
+
+        for m in self.measurements:
+            if isinstance(m.obs, list):
```

Comment:
I don't think `m.obs` would ever be a list.  It should only be either `None` or an `Operator`.